### PR TITLE
bug: Don't send non-priority messages to sentry

### LINKED ISSUE #830: Don't send bad APNS token exceptions to Sentry
Per Sentry issue:
https://sentry.prod.mozaws.net/operations/autopush-prod/issues/384162/

These should be logged, but don't need to be sent into Sentry.

### PR DESCRIPTION
**Pull request by jrconlin**
closes #830

### CODE REVIEW

File: autopush/main.py [hunk 1]
```diff
@@ -532,6 +533,9 @@ def connection_main(sysargs=None, use_files=True):
     resource = DefaultResource(WebSocketResource(factory))
     resource.putChild("status", StatusResource())
     site_factory = Site(resource)
+    # Silence starting/stopping messages
+    site_factory.noisy = args.debug
+    site.noisy = args.debug
```
Comment (pjenvey) [inline autopush/main.py]: Looks good but let's also set noisy on the other 2 cyclone.web.Applications in this file as well

Comment (jrconlin) [inline autopush/main.py]: fair enough. I thought I was being overly assertive with the base class, but 100% OK with being even more assertive.

Comment (pjenvey) [inline autopush/main.py]: oops, this assignment's now a dupe of the one on line #511

Comment (pjenvey) [inline autopush/main.py]: oops, this assignment's now a dupe of the one on line #511

**Pull request merged**